test/assert.rb

```diff
@@ -20,8 +20,10 @@ def t_print(*args)
 def assertion_string(err, str, iso=nil, e=nil, bt=nil)
   msg = "#{err}#{str}"
   msg += " [#{iso}]" if iso && iso != ''
-  msg += " => #{e.cause}" if e && e.respond_to?(:cause)
-  msg += " => #{e.message}" if e && !e.respond_to?(:cause)
+  if e
+    m = e.respond_to?(:cause) ? e.cause : e.message
+    msg += " => #{m}" if m && !m.empty?
```

Comment:
I have no objection to removing unneeded `=>` but this patch does not seem to remove it. Am I missing something?

Reply:
My expression might not be good.

It is my intention to output `=>` only when it involves exception message.

### With message

    Skip: GC in rescue => backtrace isn't available (core)

### Without message

    Skip: Method#source_location (mrbgems: mruby-method)

An intention when a mrbgem name is outputted in 280bad6 is the same, I think.

Reply:
Got it. Thank you for your explanation.